src/pages/Detail/StoreImgBox/StoreImgBox.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import './StoreImgBox.scss';
+
+class StoreImgList extends React.Component {
+  state = {
+    slideSpot: 0,
+  };
+
+  n = 7; //이미지 수
+  slideGap = 14;
+  slideMovingUnit = 500;
+  imgWidth = 400;
+  slideWidth = this.imgWidth * this.n + (this.n - 1) * this.slideGap;
+  hiddenedSlideWidth = this.slideWidth - window.innerWidth;
+  slideEnd;
+
+  handlePrevBtn = () => {
+    const { slideMovingUnit } = this;
+    const { slideSpot } = this.state;
+
+    if (Math.abs(slideSpot) < slideMovingUnit) {
+      this.setState({
+        slideSpot: 0,
+      });
+    } else {
+      this.setState({
+        slideSpot: slideSpot + slideMovingUnit,
+      });
+    }
+  };
+
+  handleNextBtn = () => {
+    let { hiddenedSlideWidth, slideMovingUnit } = this;
+    const { slideSpot } = this.state;
+
+    if (hiddenedSlideWidth - Math.abs(slideSpot) < slideMovingUnit) {
+      this.setState({
+        slideSpot: slideSpot - (hiddenedSlideWidth - Math.abs(slideSpot)),
+      });
+      this.slideEnd = slideSpot - (hiddenedSlideWidth - Math.abs(slideSpot));
+    } else {
+      this.setState({
+        slideSpot: slideSpot - slideMovingUnit,
+      });
+    }
+  };
+
+  render() {
+    const { handlePrevBtn, handleNextBtn } = this;
+    const { slideSpot } = this.state;
+
+    return (
+      <div className="storeImgBox">
+        {slideSpot !== 0 && (
```

Comment:
```suggestion
        {!!slideSpot && (
```
true일 때만 표현한다면 반대연산자를 두번 써주는 것도 간단해보이네요!